plugins/civchat2-paper/src/main/java/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -365,6 +447,33 @@ public void sendGroupMsg(Player sender, Group group, String message) {
                 return;
             }
         }
+        //LongSetting banSetting = instance.getCivChat2SettingsManager().getGlobalChatMuteSetting();
+
+        if (group.getName().equals(config.getGlobalChatGroupName())) {
+            // Chat filter check - block message if it contains banned words
+            if (containsBannedWord(message)) {
```

Comment:
Shouldn't this apply to every nl? Also, make this all one if statement, there's no need for two next to eachother like this.

Reply:
Fair enough, can you just fix up the if statements then?